.gitignore

```diff
@@ -9,5 +9,7 @@ trento-docs-site-ui/node_modules/
 trento-docs-site-ui/public/
 
 # Ignore docs output
-trento-docs-site/trento-docs-site-public/
-trento-docs-site/tmp_docs/
\ No newline at end of file
+trento-docs-site/build
```

Comment:
The new build directory should be ignored, but it is nice to have to check the changes!